### PR TITLE
Set Policy Driven Update path's are wrong

### PR DESCRIPTION
**Pull request by GrischaE1**
All Updates SetPolicyDrivenUpdateSource path's are wrong - there needs an "Updates" added to the settings name.
Verified under 21H2 -> HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\PolicyManager\default\Update

### CODE REVIEW

**Comment by denisebmsft** [comment]
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10617

File: windows/client-management/mdm/policy-csp-update.md [hunk 1]
```diff
@@ -3508,9 +3508,9 @@ The table below shows the applicability of Windows:
...
 If you configure this policy, also configure the scan source policies for other update types:
-- SetPolicyDrivenUpdateSourceForFeature
-- SetPolicyDrivenUpdateSourceForQuality
-- SetPolicyDrivenUpdateSourceForOther
+- SetPolicyDrivenUpdateSourceForFeatureUpdates
+- SetPolicyDrivenUpdateSourceForQualityUpdates
+- SetPolicyDrivenUpdateSourceForOtherUpdates
 
 >[!NOTE]
 >If you have not properly configured Update/UpdateServiceUrl correctly to point  your WSUS server, this policy will have no effect.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:3516]:
```suggestion
>If you have not properly configured Update/UpdateServiceUrl correctly to point to your WSUS server, this policy will have no effect. 
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 2]
```diff
@@ -3566,9 +3566,9 @@ The table below shows the applicability of Windows:
...
 If you configure this policy, also configure the scan source policies for other update types:
-- SetPolicyDrivenUpdateSourceForQuality
-- SetPolicyDrivenUpdateSourceForDriver
-- SetPolicyDrivenUpdateSourceForOther
+- SetPolicyDrivenUpdateSourceForQualityUpdates
+- SetPolicyDrivenUpdateSourceForDriverUpdates
+- SetPolicyDrivenUpdateSourceForOtherUpdates
 
 >[!NOTE]
 >If you have not properly configured Update/UpdateServiceUrl correctly to point  your WSUS server, this policy will have no effect.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:3574]:
```suggestion
>If you have not properly configured Update/UpdateServiceUrl correctly to point to your WSUS server, this policy will have no effect. 
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 3]
```diff
@@ -3624,9 +3624,9 @@ The table below shows the applicability of Windows:
...
 If you configure this policy, also configure the scan source policies for other update types:
-- SetPolicyDrivenUpdateSourceForFeature
-- SetPolicyDrivenUpdateSourceForQuality
-- SetPolicyDrivenUpdateSourceForDriver
+- SetPolicyDrivenUpdateSourceForFeatureUpdates
+- SetPolicyDrivenUpdateSourceForQualityUpdates
+- SetPolicyDrivenUpdateSourceForDriverUpdates
 
 >[!NOTE]
 >If you have not properly configured Update/UpdateServiceUrl correctly to point your WSUS server, this policy will have no effect.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:3632]:
```suggestion
>If you have not properly configured Update/UpdateServiceUrl correctly to point to your WSUS server, this policy will have no effect. 
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 4]
```diff
@@ -3682,9 +3682,9 @@ The table below shows the applicability of Windows:
...
 If you configure this policy, also configure the scan source policies for other update types:
-- SetPolicyDrivenUpdateSourceForFeature
-- SetPolicyDrivenUpdateSourceForDriver
-- SetPolicyDrivenUpdateSourceForOther
+- SetPolicyDrivenUpdateSourceForFeatureUpdates
+- SetPolicyDrivenUpdateSourceForDriverUpdates
+- SetPolicyDrivenUpdateSourceForOtherUpdates
 
 >[!NOTE]
 >If you have not properly configured Update/UpdateServiceUrl correctly to point your WSUS server, this policy will have no effect.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:3690]:
```suggestion
>If you have not properly configured Update/UpdateServiceUrl correctly to point to your WSUS server, this policy will have no effect. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @GrischaE1,  not related, but could you kindly also commit the suggested changes to correct the typos? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — APPROVED** [review]

**Pull request merged**